File: 598 Range Addition II.java
    public int maxCount(int m, int n, int[][] ops) {
        int minX = Integer.MAX_VALUE, minY = Integer.MAX_VALUE;
        for (int[] op : ops) {
            minX = Math.min(op[0], minX);
            minY = Math.min(op[1], minY);
        }
        
        if (minX == Integer.MAX_VALUE && minY == Integer.MAX_VALUE) return m * n;
        return minX * minY;
    }

File: 598 Range Addition II.go
func maxCount(m int, n int, ops [][]int) int {

	if len(ops) == 0 {
		return m * n
	}
	a := m
	b := n

	for _, v := range ops {
		if v[0] < a {
			a = v[0]
		}
		if v[1] < b {
			b = v[1]
		}
	}

    return a * b
    
}
